logicalplan/plan.go

```diff
@@ -10,6 +10,10 @@ import (
 	"github.com/prometheus/prometheus/promql/parser"
 )
 
+var NoOptimizers = []Optimizer{}
```

Comment:
Do we need to export these two? :thinking:

Reply:
I thought we could export `NoOptimizers` as a helper to disable all optimizers, including default ones. I guess the `optimizer` interface can be private if that's what you had in mind.

Reply:
I think it's fine for now 👍🏽